modules/ROOT/pages/parallel-foreach-scope.adoc

```diff
@@ -34,7 +34,7 @@ This XML example adds to every element in the collection the string `"-result"`:
 
 ----
 
-Every route starts with the same initial variable values. Modification to a variable within a specific route does not affect other routes. So if a variable is added or modified in one route, then, after aggregation, the value is defined by that route. If a variable is added or modified by more than one route, the value is added to a list of all the values defined for that variable within all the routes, for example:
+Every route starts with the same initial variable values. Modification to a variable within a specific route does not affect other routes. The set of variables (and their values) will stay the same as before the execution of the Parllel ForEach. 
```

Comment:
I don't think that documentation for this component should be written in terms of routes. For the user there are no routes here, just a scope with a set of components inside. It should be rephrased in terms of "variables set while processing one element will not be visible while processing the next one. All those variable changes will not be available outside of the parallel-foreach block. All variables will be returned to the values they had before entering the scope. And so on. Also add an example showing how to use the payload to communicate the information that would traditionally go into variables.